src/pages/patientView/trialMatch/TrialMatchTableUtils.ts

```diff
@@ -85,9 +85,27 @@ export function groupTrialMatchesByAgeNumerical(armGroup: ITrialMatch[]): IClini
         clinicalGroupMatch.notMatches = positiveAndNegativeMatches.notMatches;
         return clinicalGroupMatch;
     });
+    if (matches.length > 1) {
+        return mergeClinicalGroupMatchByAge(matches);
+    }
     return matches;
 }
 
+// Merge clinical matches by age when they have the same trialOncotreePrimaryDiagnosis but different trialAgeNumericals.
```

Comment:
@JiaoJiao123 does this one have a test?

Reply:
Not yet. I'll create a test for the function. Btw, I'm @victoria34. You know, I don't want to miss any messages. Thanks

Reply:
Added test for mergeClinicalGroupMatchByAge()